.github/CONTRIBUTING.md

```diff
@@ -123,6 +123,31 @@ This can be very helpful when you've made changes to only certain parts of the p
 
 We recognize linting can be annoying - if you do not want to do it, please contact a project maintainer, and they can help you with it. We do not want this to be a blocker for good code getting contributed.
 
+### Spellcheck
+
+Spellchecking for this project is done via [codespell](https://github.com/codespell-project/codespell).
```

Comment:
note that `codespell` is not really a full blown spellchecker - it is rather a "finder of common typos", so many misspelled words could exist, and on contrary -- `codespell` could alert about some correct, but rarely used words, which are more typically appearing as typos.  So may be make it explicit here:

```suggestion
Spellchecking for this project is done via [codespell](https://github.com/codespell-project/codespell).
Note that `codespell` finds common typos, so could have false-positive (correctly spelled but rarely used) and false-negatives (not finding misspelled) words.
```